packages/run-protocol/src/vpool-xyk-amm/pool.js

```diff
@@ -114,8 +114,18 @@ const helperBehavior = {
     }
     zcfSeat.exit();
     updateUpdaterState(updater, pool);
+    facets.helper.updateMetrics();
     return 'Added liquidity.';
   },
+  updateMetrics: ({ state, facets }) => {
+    const payload = harden({
+      Central: facets.pool.getCentralAmount(),
+      Secondary: facets.pool.getSecondaryAmount(),
+      Liquidity: state.liqTokenSupply,
```

Comment:
extant metrics are `camelCase`. Open to changing that but I think we should try to be consistent.

Reply:
These match the keywords, which are (uniquely) initial cap. I think it's worth supporting that exception, without relaxing camelCase as a general rule.